packages/backend/src/modules/user/user.service.ts

```diff
@@ -70,4 +83,107 @@ export default class UserService {
       _id: { $ne: exceptUser },
     });
   }
+
+  public async resetPasswordRequest(email: string): Promise<boolean> {
+    const user = await this.findByEmail(email);
+    if (!user) {
+      throw new UserInputError(`User with this email does not exist`);
```

Comment:
This is information disclosure which a potential attacker can use to narrow down the brute-force attack. Replace with "User with such email/password does not exist".